### PR TITLE
[zh-cn]: update translation of Window `scrollbars` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/window/scrollbars/index.md [hunk 1]
```diff
@@ -1,47 +1,37 @@
...
+{{APIRef()}}
 
-### 概要
+返回 `scrollbars` 对象。
 
-返回可以检查其可见性的滚动条对象。
+这是一组包含布尔类型 `visible` 属性的 `Window` 属性之一，该属性过去用于表示 web 浏览器用户界面的特定部分是否可见。
 
-### 语法
+出于隐私和互操作性的原因，如果此 `Window` 是弹出窗口，则 `visible` 属性的值为 `false`，反之为 `true`。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollbars/index.md:14]:
```suggestion
出于隐私和互操作性的原因，如果 `Window` 是弹出窗口，则 `visible` 属性的值为 `false`，反之为 `true`。
```

File: files/zh-cn/web/api/window/scrollbars/index.md [hunk 2]
```diff
@@ -1,47 +1,37 @@
...
-objRef = window.scrollbars
-```
+## 值
 
-### 示例
+包含单个属性的对象：
 
-下面的 HTML 示例展示了如何使用 window\.scrollbars 对象的 visible 属性。
+- `visible` {{ReadOnlyInline}}
+  - : 布尔属性，如果此 `Window` 是弹出窗口，则为 `false`，反之为 `true`。
```
Comment (T34-active) [inline files/zh-cn/web/api/window/scrollbars/index.md:21]:
```suggestion
  - : 布尔属性，如果 `Window` 是弹出窗口，则为 `false`，反之为 `true`。
```

File: files/zh-cn/web/api/window/scrollbars/index.md [hunk 3]
```diff
@@ -1,47 +1,37 @@
...
 ---
 
-{{ APIRef() }}
+{{APIRef()}}
 
-### 概要
+返回 `scrollbars` 对象。
 
-返回可以检查其可见性的滚动条对象。
+这是一组包含布尔类型 `visible` 属性的 `Window` 属性之一，该属性过去用于表示 web 浏览器用户界面的特定部分是否可见。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/window/scrollbars/index.md:12]:
```suggestion
这是包含布尔值 `visible` 属性的一组 `Window` 属性之一，用于表示 web浏览器用户界面的特定部分是否可见。
```

**Review by JasonLamv-t — COMMENTED** [review]
Almost good. Just need a little patch.

**Review by JasonLamv-t — APPROVED** [review]
Thank you! @T34-active :+1:

**Pull request merged**